### ISSUE TITLE
Be able to specifiy each plugin's windows version (7/10/11)

### ISSUE THREAD

**Issue by ywmaa** [robbert-vdh/yabridge#366]
### Feature description

Be able to specifiy each plugin's windows version (7/10/11)

Why ?
for some reason BBC Symphony works when winecfg is set to windows 7, meanwhile this breaks LABS! (gives unable to load their sample libraries (Error #X: Something went wrong) 
if I set the windows version to 10 or 11, LABS works, but BBC Symphony gives the unable to load their sample libraries (Error #X: Something went wrong).

I think I can workaround this by doing two wine prefixes for each and setting windows version for each, but it would be awesome if I can just do it through yabridge config, and more cleaner.

also about BBC Symphony and LABS, I tried the solution in readme many times

> If Spitfire Audio plugins like BBC Symphony Orchestra and LABS are unable to load their sample libraries (Error #X: Something went wrong), then you can try reinstalling those plugins to a new, clean Wine prefix. To avoid potential confusion, make sure to uninstall the Spitfire software along with the VST2 and VST3 plugins from your main Wine prefix first.

but the only solution for me for some reason is to change the windows version, which means I am unable to use both in the same project.

**Comment by wastee** [comment]
You can use multiple WINEPREFIX to install different sofeware and set different windows version.

**Comment by robbert-vdh** [comment]
That's indeed what Wine prefixes should be used for! While there are some environment variables that overlap some of the settings you can set through `winecfg`, plugins requiring bespoke configuration should just be using their own prefix so their configuration doesn't conflict with other plugin's setups.

**Comment by ywmaa** [comment]
Yeah, I am using wine prefixes for now and it is working, as also I mentioned in the issue that this workaround exists.

I don't actually insist on this feature and I don't mind closing the issue.

it is just a nice to have feature.
